### ISSUE TITLE
IF YOU SEE A DOCKER PROCESS RUNNING DO NOT UNINSTALL IT!!!! ASK MAYBE

### ISSUE THREAD

**Issue by gastonmorixe** [docker/dockercloud-agent#5]
Sorry, but I run this and killed my docker-machine "docker-engine" process, more over uninstallation was a pain. My server went completely down without this warning me.

**Comment by fermayo** [comment]
The installation script is meant to be run unattended, so there is no prompt to confirm the removal of the Docker Engine. When the Docker Cloud Agent is installed, the Docker Engine is also installed (by the agent) and all images and containers that already exist on the host are reused. Uninstalling the DC Agent is as easy as installing the Docker Engine package again, which will remove the DC Agent in the process.

**Comment by gastonmorixe** [comment]
I don't agree at all. If you see docker installed and well of course running, you SHOULD ask. It killed my docker-machine installation, and don't ask me why but it took me 4 hours to be able to reprovision that machine again. After instilling docker-cloudagent for some reason "docker-machine provision" stopped working. I think it had something to do with the deb sources and upstart. Your removal script is not removing what it should.  

Please. If you don't intent to make this with docker-machine or an already installation of docker (running) compatible (I thought my container will appear on the cloud seamlessly without rebooting anything , not that it will delete my docker installation right away and stop letting me using docker-machine) Think that you might be taking a SERVICE down.
